dashboard/app/serializers/api/v1/pd/enrollment_flat_attendance_serializer.rb

```diff
@@ -10,15 +10,21 @@ def school
     object.school_info&.school&.name || object.school_info&.school_name || object.school
   end
 
+  def cdo_scholarship
+    object.scholarship_status == Pd::ScholarshipInfoConstants::YES_CDO ? 'Yes' : ''
+  end
+
+  def other_scholarship
+    object.scholarship_status == Pd::ScholarshipInfoConstants::YES_OTHER ? 'Yes' : ''
+  end
+
   # Add dynamic attributes for each session's date and attendance
-  def attributes(attrs = {})
-    super(attrs).tap do |data|
+  def attributes(requested_attrs = nil, reload = false)
```

Comment:
The superclass method signature and behavior changed, hence this update. It only works correctly now if we pass attributes = `nil` to super.

Btw this serializer dynamically generating keys for `session_n_date` and `session_n_attendance` makes me kind of upset... why isn't it just a list 😭